Makefile

```diff
@@ -1,5 +1,5 @@
-REBAR_VERSION = 3.14.3-emqx-2
-REBAR = ./rebar3
+REBAR_VERSION = 3.14.3-emqx-3
+REBAR = $(CURDIR)/rebar3
 export PKG_VSN ?= $(shell git describe --tags --always)
```

Comment:
```suggestion
export PKG_VSN ?= $(shell git describe --tags --match '[0-9]*')
```

Reply:
However this may fail if it's a shallow clone. e.g. when clone a branch with `depth=1` and the HEAD commit does not have a tag.

Give we have repeated usage of this command, maybe it makes sense to create a script for it.
```
#!/bin/sh
git describe --tags --match '[0-9]*' 2>/dev/null || git describe --always
```

Reply:
Maybe we can just fill in the value of PKG_VSN with commit id?